server/config/routes.js

```diff
@@ -19,70 +19,75 @@ module.exports.routes = {
+  'get /api/sermons/get-sermon-series': 'sermons/get-sermon-series',
+  'get /api/sermons/get-service-types': 'sermons/get-service-types',
 
   // Media APIs
-  "get /api/media/get-media": "media/get-media",
+  'get /api/media/get-media': 'media/get-media',
 
   // Auth APIs
-  "post /api/auth/login": "auth/post-login",
-  "post /api/auth/login-google": "auth/post-login-google",
-  "post /api/auth/signup": "auth/post-signup",
-  "post /api/auth/signup-google": "auth/post-signup-google",
-  "post /api/auth/confirm-email": "auth/post-confirm-email",
-  "post /api/auth/verify-token/": "auth/verify-token",
-  "post /api/auth/forgot-password": "auth/post-forgot-password",
-  "post /api/auth/change-password-recovery":
-    "auth/post-change-password-recovery",
+  'post /api/auth/login': 'auth/post-login',
+  'post /api/auth/login-google': 'auth/post-login-google',
+  'post /api/auth/signup': 'auth/post-signup',
+  'post /api/auth/signup-google': 'auth/post-signup-google',
+  'post /api/auth/confirm-email': 'auth/post-confirm-email',
+  'post /api/auth/verify-token/': 'auth/verify-token',
+  'post /api/auth/forgot-password': 'auth/post-forgot-password',
+  'post /api/auth/change-password-recovery':
+    'auth/post-change-password-recovery',
 
   // User APIs
-  "get /api/users/get": "users/get-user",
-  "put /api/users/update": "users/update-user",
-  "put /api/users/delete": "users/delete-user",
-  "post /api/users/change-password": "users/post-change-password",
+  'get /api/users/get': 'users/get-user',
+  'put /api/users/update': 'users/update-user',
+  'put /api/users/delete': 'users/delete-user',
+  'post /api/users/change-password': 'users/post-change-password',
 
   // Baptism APIs
-  "post /api/baptism/create": "baptisms/create-baptism",
-  "get /api/baptism/get": "baptisms/get-baptism",
-  "put /api/baptism/update": "baptisms/update-baptism",
-  "put /api/baptism/delete": "baptisms/delete-baptism",
+  'post /api/baptism/create': 'baptisms/create-baptism',
+  'get /api/baptism/get': 'baptisms/get-baptism',
+  'put /api/baptism/update': 'baptisms/update-baptism',
+  'put /api/baptism/delete': 'baptisms/delete-baptism',
 
   // Membership APIs
-  "post /api/membership/create": "memberships/create-membership",
-  "get /api/membership/get": "memberships/get-membership",
-  "put /api/membership/update": "memberships/update-membership",
-  "put /api/membership/delete": "memberships/delete-membership",
+  'post /api/membership/create': 'memberships/create-membership',
+  'get /api/membership/get': 'memberships/get-membership',
+  'put /api/membership/update': 'memberships/update-membership',
+  'put /api/membership/delete': 'memberships/delete-membership',
+
+  // Whitelist APIs
+  'get /api/whitelist/get': 'whitelist/get-whitelist',
+  'post /api/whitelist/create': 'whitelist/post-create-whitelist',
+  'post /api/whitelist/update': 'whitelist/post-update-whitelist',
 
   // UI
-  "get /*": {
+  'get /*': {
     skipAssets: true,
```

Comment:
actually, is this the reason why we cannot access assets in our server lol, like the brp stuff have to be in images folder

Reply:
i dont think so actually lolol